### ISSUE TITLE
move cc attribution to bottom of page

### ISSUE THREAD

**Issue by gyachdav** [Rostlab/JS16_ProjectF#415]
place under copyright line

**Comment by kajo404** [comment]
Did you read the feedback on reddit? :D 

> The map under each character seems to be off. You have to scroll to the right in order to see it. Also the Creative Commons license in the footer seems to be justified, which is a bit annoying. Maybe reword it to "CC Attribution" so you don't have 3 lines? Other than that, I love the data you show. I'm not a fan of GoT, sorry I don't have much to offer. However, always a fan of quantified data & social analyses. I'm not a web dev or designer. Just a regular user passing through :P
